drivers/usb/gadget/function/f_scm.c

```diff
@@ -0,0 +1,567 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * f_scm.c -- USB Socket Control Module (SCM) function driver
+ *
+ * Copyright (C) 2018-2019 Xaptum, Inc.
+ */
+
+#include <linux/kernel.h>
+#include <linux/module.h>
+#include <linux/device.h>
+#include <linux/usb/composite.h>
+#include "u_scm.h"
+
+/*
+ * This function is a "Socket Control Module" (SCM) link. The purpose of
```

Comment:
I think all of our documentation has this wrong, but "Module" should be "Model".